components/switch/style/index.tsx

```
@@ -55,6 +55,7 @@ export default (theme: Theme) =>
       textAlign: 'center',
       alignItems: 'center',
       justifyContent: 'center',
+      zIndex: theme.switch_inner_zindex
```

ESLint

[error] Insert `,`

(prettier/prettier)